libraries/surplus/src/advanced-tests.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import S from 's-js';
+import * as Surplus from "surplus";
+import expect from "expect";
+import {
+  ComponentWithoutChildren,
+  ComponentWithChildren,
+  ComponentWithChildrenRerender,
+  ComponentWithDifferentViews,
+  ComponentWithProperties,
+  ComponentWithUnregistered,
+  ComponentWithImperativeEvent,
+  ComponentWithDeclarativeEvent
+} from "./components";
+
+
+//let _it = it;
```

Comment:
Did you mean to leave this comment?

Reply:
nope, will remove.  Was left from trying to see if I could wrap calls to `it()` in `S.root()` so as to modify the test themselves less.